.NET/Microsoft.Recognizers.Definitions.Common/Chinese/DateTimeDefinitions.cs

```diff
@@ -61,6 +61,7 @@ public static class DateTimeDefinitions
       public const string DateUnitRegex = @"(?<unit>年|个月|周|週|日|天)";
       public const string BeforeRegex = @"以前|之前|前";
       public const string AfterRegex = @"以后|以後|之后|之後|后|後";
+      public const string TimePeriodLeftRegex = @"^[.]";

```

Comment:
Don't similar situatuions exist in Chinese? Perhaps we should open an issue to create the equivalent specs in Chinese and implement support.